index.html

```diff
@@ -4813,13 +4813,45 @@ <h2>Contexts, Types, and Credential Schemas</h2>
       <section class="informative">
         <h3>Base Context</h3>
           <p>
-The base context, located at
-<code>https://www.w3.org/ns/credentials/v2</code> with a SHA-256 digest of
+The base context, located at <code>https://www.w3.org/ns/credentials/v2</code>
+with a SHA-256 digest of
 <strong><code>944167aaabd904ea9e35c98fd7e8794eb6dd42ae4666b036b171e87fc34cc7cc</code></strong>,
```

Comment:
I think we should consider also adding a hash of the JCS-canonized version of the context to eliminate any differences from whitespace, etc.

Reply:
The non-canonized hash I get if I use the current `v2` context in the repo is this:

`e41d1743c941b8003a4689e6e844b7f6ae0b9970ed81aa918bd781977bdba8b2`

The canonized hash I get is this:

`d64fce2a2f642ddf9f74a6aa01eb16efdaa822d0380d4891d310f3a1e085e671`

Neither of these matches the value here, which may just be out of date.

Reply:
Fixed in aa4d74c ... which has a different hash than the two items above.